rmw_connextdds_common/src/common/rmw_impl.cpp

```diff
@@ -221,6 +221,19 @@ rmw_connextdds_find_string_in_list(
   return false;
 }
 
+DDS_Duration_t
+rmw_connextdds_duration_from_ros_time(
+  DDS_Duration_t * const duration,
+  const rmw_time_t * const ros_time)
```

Comment:
This is not currently matching the declaration in the header:

```suggestion
DDS_Duration_t
rmw_connextdds_duration_from_ros_time(
  const rmw_time_t * const ros_time)
```

Reply:
you can also use `const rmw_time_t * ros_time`, as the second `const` isn't really useful

Reply:
Yes.

Reply:
Yes.